src/main/java/se/sundsvall/supportmanagement/integration/emailreader/EmailReaderClient.java

```diff
@@ -9,19 +8,27 @@
 import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestParam;
 
 import se.sundsvall.supportmanagement.integration.emailreader.configuration.EmailReaderConfiguration;
 
 import generated.se.sundsvall.emailreader.Email;
+import io.swagger.v3.oas.annotations.Parameter;
 
 @FeignClient(name = CLIENT_ID, url = "${integration.emailreader.url}", configuration = EmailReaderConfiguration.class)
 public interface EmailReaderClient {
 
-	@GetMapping(path = "/email", produces = APPLICATION_JSON_VALUE)
-	List<Email> getEmails(@RequestParam("municipalityId") final String municipalityId, @RequestParam("namespace") final String namespace);
+	@GetMapping("/{municipalityId}/email/{namespace}")
+	List<Email> getEmails(
+		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281")
```

Comment:
Swagger annotations are only useful on the REST-controllers (since that will affect the service API), not the clients.
(See comment on PR for email-reader)